package.json

```diff
@@ -8,14 +8,14 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^3.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
Why did you add `--openssl-legacy-provider`?

Reply:
Without this option, it gives me the below error
![image](https://github.com/mayurkamaliya/netflixroulette/assets/59191958/bba15ae6-d8a5-4235-83ce-2804c4165123)

Reply:
ok, let's proceed with your solution.